src/compass/s1_geocode_stack.py

```diff
@@ -70,6 +70,9 @@ def create_parser():
     optional.add_argument('-m', '--metadata', action='store_true',
                           help='If flat is set, generates radar metadata layers for each'
                                ' burst stack (see rdr2geo processing options)')
+    optional.add_argument('--unzipped', action='store_true',
```

Comment:
I am a bit confused here (I might need a bit more context). I have never unzipped my SAFE file when using COMPASS and specifically stack processor. What is the aim of this option?

Reply:
The aim is for quicker runtime when geocoding many bursts from one granule. Here's the numbers I got when I checked geocoding one burst using the .zip file vs the unzipped .SAFE folder.

| Threads | Zipped runtime (s) | Unzipped runtime (s) |
| ------- | ------------------ | -------------------- |
| 8       | 50                 | 39                   |
| 16      | 34                 | 22                   |
| 32      | 25                 | 14                   |
| 48      | 21                 | 10                   |


The `Threads` were set by doing, e.g., `$ OMP_NUM_THREADS=16 s1_cslc.py test-zipped.yaml`

Geocoding all the bursts in a stack of 20 dates means that if you can us 48 threads, it'll take about `21 * 30 * 30 = 210` minutes using the .zip, and 100 minutes using the .SAFE.  
The 20 dates took 3.5 - 4 minutes to unzip the VV polarization images/metadata.

Reply:
Thanks for the explanation. Does the unzipped runtime include the time to unzip the SAFE file itself? I would expect that the unzipped runtime just includes the time to run the CSLC-S1 workflow on an unzipped SAFE file.

Reply:
Sorry i smushed that part together up there:

The 20 dates took 3.5 - 4 minutes to unzip unto SAFE files. I selectively unzipped the VV polarization images/metadata only, not the VH.